### PR TITLE
Kestrel bays

### PR DESCRIPTION
**Pull request by Zitchas**
**Content**
The Kestrel is currently situated in sort of an odd spot: It requires a massive combat rating to  unlock, one that most players will not encounter until well into the Wanderer campaign unless they spend a huge amount of time fighting stuff. Beyond that, it is a mission that will trigger on only a single planet that doesn't have much in the way of story or mission content around. All this to unlock an Easter egg ship that - while it does have some fun corner cases - is very weak and under-powered compared to what the player will have available at that point. On the flip side, it *is* possible for the player to get it before they have completed any campaign or met any aliens, so it can't 

On a parallel note, the original Kestrel in Escape Velocity had a pair of Lightning Fighter Bays, which carried ships that look and feel remarkably like slightly tougher versions of the Berserker. This PR doesn't address the Lightning, but it would be worth considering in the future.

## Summary
This PR adds a pair of fighter bays to the Kestrel and to MZ's person ship. Nothing more, nothing less. The bays are both "under" the wings, one on either side.

##Related
This PR is related to the discussion that has been ongoing surrounding the Kestrel, most recently in the live-stream and PR #4961 . Given that this is a broad change to all the Kestrels and not specific to any one variant, I've created this so it can be added separately from the discussion regarding the Shield variant in particular.

## PR Checklist 
- ~~[ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified~~
- ~~[ ] I uploaded the necessary image, blend, and texture assets here: {{insert link to assets}}~~
- ~~[ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}~~

### CODE REVIEW

**Comment by ravenshining** [comment]
I think in the future, it might be nice to add some bays *inside* the wing such that you'd see them in the thumbnail, but this is great as it is!

**Comment by Zitchas** [comment]
Modifying the Kestrel to show the bays in the future is a possibility. In the original Kestrel the bays were actually located in the central shaft, about halfway to the tip. Here's a scanner report from the original game. This also gives a bit of the scale that Matt Burch originally intended for the Kestrel.

vgvqPmxRfutmPcgbihXe8SAUfwO998G05P8PmTjT13Ag5bh_3si7rKio0T8qtODtzNq-aNPX3EAzGY-SQVc7CyidUv4a6NRvdCHQ0tnygpTnNMKDSqABn80J

Many artistic variations that I've seen since then have moved it back to underneath the main body between the rear fins.

**Comment by Zitchas** [comment]
I have the two finches added to MZ - Now I'm just waiting for him to show up. I'm currently running through the entire Remnant mission set with the new Korath stuff, but I've patched in the changes to the Kestrel & MZ (with the frequency massively boosted) to see how things turn up.

File: data/persons.txt [hunk 1]
```diff
@@ -74,6 +76,10 @@ person "Michael Zahniser"
 		explode "medium explosion" 80
 		explode "large explosion" 120
 		explode "huge explosion" 60
+	ship "Finch" "Finch"
+		"never disabled"
```
Comment (ravenshining) [inline data/persons.txt:80]: His finches need some pizzazz.  Going with the theme...
```suggestion
		"never disabled"
		add attributes
			"weapon capacity" 22
			"outfit space" 140
			"shields" 900
			"hull" 1800
		outfits
			"Electron Beam" 2
			
			"Fuel Pod"
			"Liquid Nitrogen Cooler"
			"Large Radar Jammer"
			"Ramscoop"
			"S-270 Regenerator"
			
			"LP036a Battery Pack"
			"Fission Reactor"
			
			"A125 Atomic Steering"
			"A120 Atomic Thruster"
			"Ionic Afterburner"
		gun -7 -14 "Electron Beam"
		gun 7 -14 "Electron Beam"
```

File: data/persons.txt [hunk 2]
```diff
@@ -74,6 +76,10 @@ person "Michael Zahniser"
...
+	ship "Finch" "Finch"
+		"never disabled"
```
Comment (ravenshining) [inline data/persons.txt:82]:
```suggestion
		"never disabled"
		add attributes
			"weapon capacity" 22
			"outfit space" 140
			"shields" 900
			"hull" 1800
		outfits
			"Electron Beam" 2
			
			"Fuel Pod"
			"Liquid Nitrogen Cooler"
			"Large Radar Jammer"
			"Ramscoop"
			"S-270 Regenerator"
			
			"LP036a Battery Pack"
			"Fission Reactor"
			
			"A125 Atomic Steering"
			"A120 Atomic Thruster"
			"Ionic Afterburner"
		gun -7 -14 "Electron Beam"
		gun 7 -14 "Electron Beam"
```

**Comment by ravenshining** [comment]
I did some testing by adding a bunch of MZ clones to persons.txt and hitting capslock while flying a cloaked Archon in a Pirate system.  The Finches also bore his name.

**Comment by tehhowch** [comment]
> 
> 
> I did some testing by adding a bunch of MZ clones to persons.txt and hitting capslock while flying a cloaked Archon in a Pirate system. **The Finches also bore his name.**

fixed the bold bit for 0.9.13.

As far as adding bays to the purchasable Kestrel, I'm neither for nor against.

File: data/persons.txt [hunk 3]
```diff
@@ -74,6 +76,54 @@ person "Michael Zahniser"
 		explode "medium explosion" 80
 		explode "large explosion" 120
 		explode "huge explosion" 60
+	ship "Finch" "Finch"
+		"never disabled"
+		add attributes
+			"weapon capacity" 22
```
Comment (ravenshining) [inline data/persons.txt:82]: to resolve the failed check:
```suggestion
			"weapon capacity" 22
			"engine capacity" 22
```

File: data/persons.txt [hunk 4]
```diff
@@ -74,6 +76,54 @@ person "Michael Zahniser"
...
+			
+			"A125 Atomic Steering"
+			"A120 Atomic Thruster"
+			"Ionic Afterburner"
+		gun -7 -14 "Electron Beam"
+		gun 7 -14 "Electron Beam"
+	ship "Finch" "Finch"
+		"never disabled"
+		add attributes
+			"weapon capacity" 22
```
Comment (ravenshining) [inline data/persons.txt:106]: to resolve the failed check:
```suggestion
			"weapon capacity" 22
			"engine capacity" 22
```

File: data/persons.txt [hunk 5]
```diff
@@ -74,6 +76,54 @@ person "Michael Zahniser"
...
+			"Large Radar Jammer"
+			"Ramscoop"
+			"S-270 Regenerator"
+			
+			"LP036a Battery Pack"
+			"Fission Reactor"
+			
+			"A125 Atomic Steering"
+			"A120 Atomic Thruster"
+			"Ionic Afterburner"
```
Comment (ravenshining) [inline data/persons.txt:124]: alternatively, nix the afterburner
```suggestion
```

File: data/persons.txt [hunk 6]
```diff
@@ -74,6 +76,54 @@ person "Michael Zahniser"
...
```
Comment (ravenshining) [inline data/persons.txt:100]: alternatively, nix the afterburner
```suggestion
```

**Comment by Zitchas** [comment]
OK, I just tested this using an exe built from the 0.9.13 branch, and the two finches are both named "finch" - so only MZ's kestrel itself actually has his name. 

So unless we want some sort of special name for them, I'd say this is good to go for 0.9.13 merge. (Personally, I don't know MZ well enough to know what he'd want his to fighters to be named. I'd be half inclined to call them "Vapour" and "Git", representing his two tools - Steam and Github. Or perhaps "Blender" and "Gimp")

**Comment by kestrel1110** [comment]
> OK, I just tested this using an exe built from the 0.9.13 branch, and the two finches are both named "finch" - so only MZ's kestrel itself actually has his name.
> 
> So unless we want some sort of special name for them, I'd say this is good to go for 0.9.13 merge. (Personally, I don't know MZ well enough to know what he'd want his to fighters to be named. I'd be half inclined to call them "Vapour" and "Git", representing his two tools - Steam and Github. Or perhaps "Blender" and "Gimp")

I'm all for it!

**Pull request merged**